File: Linguagem C/Vetores_Matrizes/Ex_001.c
//1) Faça um programa que leia um vetor de char com 8 posições e, a seguir, mostre os valores lidos em
//ordem inversa.
#include <stdio.h>

int main() {
    char vetor[8];
    int i;

    for (i = 0; i < 8; i++) {
        scanf(" %c", &vetor[i]);
    }
    for (i = 7; i >= 0; i--) {
        printf("%c ", vetor[i]);
    }
    printf("\n");
    
    return 0;
}

File: Python/006_Lista_vetores_matrizes/Ex_001.py
"""
1) Faça um programa que leia um vetor de char com 8 posições e, a seguir, mostre os valores lidos em
ordem inversa.
"""
vetor = []

for i in range(8):
    char = input(f"Digite o caractere{i+1}: ")
    vetor.append(char)

print("Valores em ordem inversa: ")
for i in range (7, -1, -1):
    print(vetor[i], end= " ")